code/datums/components/crafting/tailoring.dm

```diff
@@ -114,6 +114,16 @@
 				  /obj/item/stack/cable_coil = 5)
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/sunhudscience
+	name = "Science Sunglasses"
+	result = /obj/item/clothing/glasses/science/sciencesun
+	time = 20
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/science = 1,
+				  /obj/item/clothing/glasses/sunglasses = 1,
+				  /obj/item/stack/cable_coil = 5)
```

Comment:
Seems to be 2 spaces at the start of these 2 things here

Reply:
that's how all crafting recipes are done, I just copied beer googles recipe